benchexec/benchexec.py

```diff
@@ -152,6 +152,13 @@ def create_argument_parser(self):
                           metavar="N",
                           help="Limit each run of the tool to N CPU cores (-1 to disable).")
 
+        parser.add_argument("-s", "--allowedCores", dest="coreset",
+                            default=None,
+                            metavar="n",
+                            nargs='+',
+                            type=int,
+                            help="Limit each run of the tool to a subset of particular CPU cores.")
```

Comment:
Can you write "Limit the set of cores BenchExec will use for all runs" instead, please? The current description sounds like one run will be able to use all the given cores.